specs/vep-2272-complete-data-job-configuration-persistence/README.md

```diff
@@ -95,75 +95,113 @@ There would be no changes to the public APIs.
 
 ## Detailed design
 
-### Database data model changes
+### Kubernetes Cron Jobs Synchronizer
+
+To enhance data integrity and synchronization between a database and Kubernetes, an asynchronous process will be
+implemented. This process will leverage Spring Scheduler and Scheduler Lock, similar to the
+[DataJobMonitorCron.watchJobs()](https://github.com/vmware/versatile-data-kit/blob/main/projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/monitoring/DataJobMonitorCron.java#L84).
+
+The overall purpose of this process is to iterate through the data job deployment configurations stored in the database
```

Comment:
How often will these iterations/checks be scheduled? Please clarify.

Reply:
Incorporated one sentence about that.